libs/@hashintel/component-library/src/components/Bar/bar.tsx

```diff
@@ -0,0 +1,93 @@
+import { useId } from "react";
+
+import { Filter } from "../../lib/flexible-filter";
+import { CONVEX } from "../../lib/surface-equations";
+import { useMotionResizeObserver } from "../../lib/use-motion-resize-observer";
+
+export type BarProps = React.PropsWithChildren<{
```

Comment:
I'm not 100% sure `Bar` is the correct name for this component versus something like `Panel`,`BackgroundPanel`, or `Underlay`. cc @CiaranMn for thoughts